### PR TITLE
Project setup: mention 'compiledb' as an alternative to 'bear' on Windows

### PR DESCRIPTION
**Pull request by HighCommander4**

### CODE REVIEW

**Comment by HighCommander4** [comment]
We get a lot of questions from Windows users for whom `bear` does not work, so I figured it's worth mentioning this.

File: installation.md [hunk 1]
```diff
@@ -357,6 +357,9 @@ compile_commands.json file by recording a complete build.
 
 For a `make`-based build, you can run `make clean; bear -- make` to generate the
 file (and run a clean build!).
+
+On Windows, a tool similar to Bear called [compiledb](https://github.com/nickdiego/compiledb)
```
Comment (hokein) [inline installation.md:361]: This tool doesn't seem to be actively maintained, the last commit was 5 years ago...

Comment (HighCommander4) [reply installation.md:361]: That's a good point.

Do you know of any other tools we could recommend to Windows users who are using `make` or other build systems without explicit `compile_commands.json` generation support?

Comment (hokein) [reply installation.md:361]: I don’t have a better suggestion. I guess it's still worth mentioning (assuming it still works today). It's better than having nothing.

Comment (HighCommander4) [reply installation.md:361]: Though I haven't tried it myself (and I don't have a Windows environment to do so), users that I've recommended it to have reported it working.

Comment (hokein) [reply installation.md:361]: Then I have no further concerns.

**Review by hokein — APPROVED** [review]

**Pull request merged**